.dockerignore

```diff
@@ -0,0 +1,107 @@
+#
```

Comment:
This file is also available in https://github.com/apache/airflow/pull/4936/

Reply:
That's correct @gerardo. Unfortunately if you have dependent PRs from another fork, you cannot base it on one-another - that's limitation of Github. 

This is one change split into three commits and Part 1 is commit 1, Part 2 is commit 1 + commit 2, Part 3 is commit 1 + commit 2 + commit 3.  Until the previous is merged, I have to base my PR on master, which means that previous commits are included. What I suggested in the PR description (but it's not readily visible) is to review the last commit only. Github supports it. You can do it easily by clicking at the commit above - and then you will be able to see only changes from the last commit.

Reply:
I will make it more obvious in the description of the PR